var/spack/repos/builtin/packages/ack/package.py

```diff
@@ -16,6 +16,8 @@ class Ack(Package):
     homepage = "http://beyondgrep.com/"
     url      = "http://beyondgrep.com/ack-2.14-single-file"
 
+    version('3.50', sha256='fd0617585b88517a3d41d3d206c1dc38058c57b90dfd88c278049a41aeb5be38', expand=False, url='http://beyondgrep.com/ack-v3.5.0')
```

Comment:
`https` seems to be working too, so we should use that if there's no compelling reason to do otherwise

Reply:
@robert-mijakovic :point_up:

Reply:
Can you tell me why it tries to parse extension for a file that doesn't have it?
==> [2021-07-20-13:17:41.190047] MirrorError: Unable to parse extension from https://beyondgrep.com/ack-v3.5.0
How to suppress this from happening?

Reply:
I get a different hash:
```
$ sha256sum ack-v3.5.0 
6870d3c90691c3c4a9ec2ae69880e85c5188aa57adeeca2a794b477e034b989f  ack-v3.5.0
```
and it seems there's also ack v3.6.0 out now. Closing this PR, feel free to open a new one.